### PR TITLE
Restructuring

### PR DESCRIPTION
**Pull request by Richard-ET**
Hey @marc-tonsen Going through your new documentation, I'll note here some of the things I spot. Also just talked with @N-M-T about how the docs should fit with the Museum Study, and I think I have a good idea now how to frame it and where it should go.

### CODE REVIEW

File: src/invisible/getting-started/ecosystem-overview.md [hunk 1]
```diff
@@ -4,4 +4,57 @@ description: TODO
...
+Once you have made a recording using your Pupil Invisible glasses, the next step is to transfer it off of the phone for further processing.
+We will introduce the different available options in the article alongside a couple other key components.
+
+## Pupil Cloud
+Pupil Cloud is our cloud-based storage and analysis platform and is the recommended tool for handling your Pupil Invisible recordings.
+It makes it easy to store all your data securely in one place and it offers a variety of options for analysis.
+
+<div class="pb-4" style="display:flex;justify-content:center;">
+  <v-img
+    :src="require('../../media/cloud/imgs/cloud-capture-store-analyze.jpg')"
```
Comment (ClaraKuper) [inline src/invisible/getting-started/ecosystem-overview.md]: This image could be clearer with titles: while it's obvious that the first image - "capture" - shows the glasses, the difference between "upload" (Companion App) and "analyze" (Cloud) might be less clear. I suggest adding titles like "PI Glasses", "PI Companion App", and "Pupil Cloud" to the images.

File: src/invisible/getting-started/ecosystem-overview.md [hunk 2]
```diff
@@ -4,4 +4,57 @@ description: TODO
...
+::: tip
+Note that without uploading to Pupil Cloud, the 200 Hz gaze signal is not available in a recording. Instead, only the real-time gaze signal computed by the phone is available, which is limited to ~65 Hz (using a OnePlus 8 phone).
+:::
+
+
+## Also worth checking out...
+There are a number of other tools available for processing Pupil Invisible recordings or for making data acquisition easier. Here we introduce a couple more key components.
+
+### 3rd Party Analysis Software
+A range of software integrations is available for Pupil Invisible recordings, including compatibility with e.g. the [iMotions]() and [Bliskshift]() software. You can find a full list of available software integrations [here]().
```
Comment (ClaraKuper) [inline src/invisible/getting-started/ecosystem-overview.md]: Typo: Blickshift

**Comment by ClaraKuper** [comment]
On this page: https://docs-staging.pupil-labs.com/invisible/how-tos/applications/gaze-metrics-in-aois/
there's a typo in the title ("gaze metrics in on AOIs")

File: src/invisible/explanation/data-streams.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+</div>
+
+The gaze data is output in pixel space of the scene camera image, which has a resolution of 1088x1080px. The origin is in the top-left corner of the image.
+
+You can find a high-level description as well as a thorough evaluation of the accuracy and robustness of the gaze estimation algorithm in our [white paper](https://arxiv.org/pdf/2009.00508).
+
+## Fixations
+The two primary types of movement the eye is performing are fixations and saccades. During fixation, the eye is fixated on a specific point in the environment. A saccade is a very quick movement where the eye is jumping from one fixation to the next. Properties like the fixation duration are correlated with cognitive processes like e.g. cognitive load.
+
+Fixations are calculated automatically in Pupil Cloud after uploading a recording. They are available as part of all enrichment downloads. The downloads for gaze mapping enrichments ([Reference Image Mapper](/invisible/explanation/enrichments/#reference-image-mapper), [Marker Mapper](/invisible/explanation/enrichments/#marker-mapper)) also include "mapped fixations".
```
Comment (Richard-ET) [inline src/invisible/explanation/data-streams.md:50]: Not strictly ALL enrichments, as Gaze Overlay (if it remains an enrichment) has no associated .csv, and overlays gaze, not fixations

**Pull request merged**